.gitattributes

```diff
@@ -18,6 +18,8 @@ CONTRIBUTING.md          ocaml-typo=missing-header
 INSTALL                  ocaml-typo=missing-header
 LICENSE                  ocaml-typo=non-printing,missing-header
 
+Makefile.shared          ocaml-typo=long-line
```

Comment:
I've changed my added lines in Makefile.shared to be less than 81 chars, but L20 in Makefile.shared was already longer (which might be why I updated in .gitattributes in the first place!)